### DISCUSSION TITLE
Difference with ordered-parallel-iterator

### DISCUSSION THREAD

**Discussion by nyurik** [dpc/pariter#5] [Q&A]
Hi, thanks for creating pariter!  I was researching this topic, and saw that there is another, more downloaded but less maintained crate called [ordered-parallel-iterator](https://crates.io/crates/ordered-parallel-iterator). Do you know if its similar, or how does pariter differs from it?  I see that it has a bit of a different API. I'm sure other devs may have the same question, would be great if the README included comparison with the other similar project. Thanks again!!!

**Comment by dpc** [comment]
I didn't like the API. I also don't understand the implementation (I would have to dig deeper than I wanted to).
